### PR TITLE
Added info about `RepoUpdateBench` into readme

### LINKED ISSUE #98: Benchmarks
Let's create benchmarks for debian adapter, it seems that we need two measure 
- index merge (`MultiPackages.Unique.merge()` method)
- repository update (`Debian.Asto` methods)

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #98 
Added information about `RepoUpdateBench` into readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,11 +140,20 @@ Here is general algorithm to run benchmarks:
  4. Run benchmarks with `env BENCH_DIR=/tmp/debian-test java -cp "target/benchmarks.jar:target/classes/*:target/dependency/*" org.openjdk.jmh.Main BenchToRun`, 
  where `/tmp/debian-test` is a directory with resources for tests, `BenchToRun` is benchmark class name.
 
-There are several benchmarks in debian-adapter: `IndexMergeBench` to test indexes merging and other. 
+There are several benchmarks in debian-adapter: `IndexMergeBench` to test indexes merging and 
+`RepoUpdateBench` for generation of repository indexes test.
```
Comment (genryxy) [inline README.md:144]: @olenagerasimova probably to add internal links on sections with these benchmarks

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one suggestion

**Comment by olenagerasimova** [comment]
@genryxy thanks, I've used references (not sure what you meant by `internal links`)

**Review by genryxy — APPROVED** [review]
@olenagerasimova I've meant internal links on sections in `README.md`, although the path to the classes is also appropriate

**Pull request merged**